src/sentry/static/sentry/app/views/settings/projectAlerts/ruleDetailsNew/index.tsx

```diff
@@ -84,14 +84,14 @@ class RuleDetails extends AsyncView<Props, State> {
                     'metric',
                     t('Metric Alert'),
                     t(
-                      'Alert on conditions defined over all events in the project. For example, more than 10 users affected by signup-page errors, database errors exceed 10 per minute, errors seen by our largest customers exceed 500 per hour.'
+                      'Metric alerts allow you to filter and set thresholds on errors. They can be used for high-level monitoring of patterns, or fine-grained monitoring of individual events.'
```

Comment:
"Metric alerts allow you to filter and set thresholds on errors"

Should errors be "events"? Because we could allow them to filter and set thresholds on things that aren't errors, as well